Makefile.defs

```diff
@@ -89,16 +90,45 @@ GO_TAGS_FLAGS += osusergo
 # RACE is specified.
 GOTEST_COVER_OPTS =
 
+CODEOWNERS_PATH_EVAL := $(shell echo $(wildcard $(ROOT_DIR)/*OWNERS*) | tr ' ' ',')
```

Comment:
One more change to make: Add `TESTOWNERS` file based on `CODEOWNERS` (likely the version from the v1.17 branch point from `main`.)